Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -102,6 +106,10 @@
     sprite: Clothing/OuterClothing/Coats/labcoat_chem.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Coats/labcoat_chem.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Acidic: 0.75
```

Comment:
why does chem labcoat have the least acid resistance of any labcoat?

Reply:
It has the most resistance of the lab coats as I understand it.  That's what testing shows and what the tooltips say.  The 0.75 is multiplied by the incoming damage, reducing the damage by 25%.  The other labcoats are 0.90 which is only a 10% reduction.

Reply:
oh yeah im idiot